hooks/nrpe_helpers.py

```diff
@@ -888,3 +882,43 @@ def set_netlinks_error():
     """Set the flag indicating a netlinks related error."""
     global NETLINKS_ERROR
     NETLINKS_ERROR = True
+
+
+def _get_cis_audit_check():
+    """Get the CIS audit step for nrpe.
+
+    When CIS audit is disabled no step is necessary.
+    """
+    if not hookenv.config("cis_audit_enabled"):
+        return {}
+
+    cmd_params = "{}-p '{}' {}".format(
+        _tailoringfile_cmd(),
+        hookenv.config("cis_audit_profile"),
+        hookenv.config("cis_audit_score"),
+    )
+    cis_audit_check = {
+        "description": "Check CIS audit",
+        "cmd_name": "check_cis_audit",
+        "cmd_exec": local_plugin_dir + "check_cis_audit.py",
+        "cmd_params": cmd_params,
+    }
+    return cis_audit_check
+
+
+def _tailoringfile_cmd():
```

Comment:
**nitpick:** 

The name is confusing. It's a action with side-effect not noun.